MathDeployment/Top/MathDeploymentPackets.xml

```diff
@@ -52,12 +52,7 @@
         <channel name="systemResources.NON_VOLATILE_TOTAL"/>
         <channel name="systemResources.NON_VOLATILE_FREE"/>
     </packet>
-
-    <packet name="SystemRes2" id="6" level="2">
-        <channel name="systemResources.FRAMEWORK_VERSION"/>
-        <channel name="systemResources.PROJECT_VERSION"/>
-    </packet>
-
+    
     <packet name="SystemRes3" id="7" level="2">
```

Comment:
Should we change this from `SystemRes3` to `SystemRes2` and change id from `7` to `6`?